python/paddle/fluid/tests/unittests/test_gru_op.py

```diff
@@ -31,7 +31,7 @@ def gru(
         is_reverse,
         act_state,
         act_gate,
-        dtype):
+        dtype='float64'):
```

Comment:
default should be float32

Reply:
Done. According to the old rule, it should be "float64". anyway, just change the default type as 'float32'.